README.md

```diff
@@ -120,7 +129,15 @@ The following describes the intent of UA client related directories:
 
 ## Testing
 
-All unit and lint tests are run using tox:
+All unit and lint tests are run using `tox`. We also use `tox-pip-version` to specify an older pip version as a workaround: we have some required dependencies that can't meet the strict compatibility checks of current pip versions.
```

Comment:
This is my attempt at a short description of why we have `tox-pip-version` based on my memory of what @lucasmoura was saying yesterday. Please chime in if its incorrect/too long/unnecessary/etc.

Reply:
I think this is a great description of the problem. Thanks @orndorffgrant